az-core/src/main/java/azkaban/utils/JSONUtils.java

```diff
@@ -277,17 +277,59 @@ private static boolean isCharSpecialUnicode(final char ch) {
 
   /**
    * Reads json file from the classpath placed in the resources folder and returns as string
+   *
    * @param filePath
    * @return String json as string
    */
-  public static String readJsonFileAsString(String filePath) {
+  public static String readJsonFileAsString(final String filePath) {
     try {
-      InputStream is = JSONUtils.class.getClassLoader().getResourceAsStream(filePath);
+      final InputStream is = JSONUtils.class.getClassLoader().getResourceAsStream(filePath);
```

Comment:
Eventhough this method is only used for unit tests, this can be used whenever required. Initially I have put this method in the unit tests only, but based on your feedback on previous RB, move this method to JSONUtils. I think we can keep this method and this utility method can be used whenever needed.